### ISSUE TITLE
Missing the mention of the ical.js author and Apache2 licence

### ISSUE THREAD

**Issue by fflorent** [naimo84/kalender-events#5]
The file [ical.ts](https://github.com/naimo84/kalender-events/blob/9990a18542bad2578ad4c3a7d153978c925fa873/src/ical.ts) seems to be taken from the [ical.js](https://github.com/peterbraden/ical.js/blob/13088047555ef95763983e4cb5c2bd48e6f36b73/ical.js) project.

As per the Apache-2 license, please mention the author and the license of the module.

**Comment by naimo84** [comment]
Hey @fflorent,

you're completly right. shame on me... I've added a special thanks sentence at the top of the readme.

I even didn't realize, that the code I copied from node-ical is already a fork of ical.js.

I have used node-ical for more than 2 years I guess. which was version 0.10.0. then I had an issue in my node-red node (https://github.com/jens-maus/node-ical/issues/169#issuecomment-984425095) and tried to fix it with 
https://github.com/jens-maus/node-ical/pull/170

BUT: while debugging the latest version of node-ical I realized that "a lot" of Code regarding to timezone was changed... though I would have had to change really much of my own code. So I decided to copy the code of node-ical and remove some of the latest Code, which Breaks my Code 😐

In long terms I'd like to switch back to node-ical. but currently I have really less free time...

tl;dr
shame on me again, I had to notice the great work of the guys quitr earlier...

**Comment by fflorent** [comment]
I see, thanks for having reacted quickly :)
